lib/ramble/ramble/cmd/workspace.py

```diff
@@ -883,15 +900,20 @@ def workspace_edit(args):
     config_file = ramble.workspace.config_file(ramble_ws)
     template_files = ramble.workspace.all_template_paths(ramble_ws)
 
-    edit_files = [config_file] + template_files
+    edit_files = [config_file]
+    edit_files.extend(template_files)
 
-    if args.config_only:
+    if args.filename:
+        edit_files = [ramble.workspace.get_yaml_filepath(ramble_ws, args.filename)]
```

Comment:
It might be worth changing the name of `get_yaml_filepath` to be more like `get_filepath` since it could be any file (like a `*.tpl` file)